platform/security/sts/security-sts-usernametokenvalidator/src/main/java/org/codice/ddf/security/validator/username/UsernameTokenValidator.java

```diff
@@ -239,69 +191,29 @@ public TokenValidatorResponse validateToken(TokenValidatorParameters tokenParame
+                    LOGGER.debug("Unable to validate user against {}" + entry.getKey(), ex);
                 }
             }
-
-            if (secToken == null) {
-                Credential credential = new Credential();
-                credential.setUsernametoken(ut);
-                //Only this section is new, the rest is copied from the apache class
-                Set<Map.Entry<String, Validator>> entries = validators.entrySet();
-                for (Map.Entry<String, Validator> entry : entries) {
-                    try {
-                        entry.getValue().validate(credential, requestData);
-                        validateTarget.setState(ReceivedToken.STATE.VALID);
-                        break;
-                    } catch (WSSecurityException ex) {
-                        LOGGER.debug("Unable to validate user against {}" + entry.getKey(), ex);
-                    }
-                }
-                if (ReceivedToken.STATE.INVALID.equals(validateTarget.getState())) {
-                    return response;
-                }
-                //end new section
+            if (ReceivedToken.STATE.INVALID.equals(validateTarget.getState())) {
+                return response;
             }
+            //end new section
 
             Principal principal = createPrincipal(ut.getName(), ut.getPassword(),
                     ut.getPasswordType(), ut.getNonce(), ut.getCreated());
 
-            // Get the realm of the UsernameToken
-            String tokenRealm = null;
-            if (usernameTokenRealmCodec != null) {
-                tokenRealm = usernameTokenRealmCodec.getRealmFromToken(ut);
-                // verify the realm against the cached token
-                if (secToken != null) {
-                    Properties props = secToken.getProperties();
-                    if (props != null) {
-                        String cachedRealm = props.getProperty(STSConstants.TOKEN_REALM);
-                        if (!tokenRealm.equals(cachedRealm)) {
-                            validateTarget.setState(ReceivedToken.STATE.INVALID);
-                            return response;
-                        }
-                    }
-                }
-            }
-
-            // Store the successfully validated token in the cache
-            if (tokenParameters.getTokenStore() != null && secToken == null
-                    && ReceivedToken.STATE.VALID.equals(validateTarget.getState())) {
-                secToken = new SecurityToken(ut.getID());
-                secToken.setToken(ut.getElement());
-                int hashCode = ut.hashCode();
-                String identifier = Integer.toString(hashCode);
-                secToken.setTokenHash(hashCode);
-                tokenParameters.getTokenStore().add(identifier, secToken);
-            }
-
             response.setPrincipal(principal);
-            response.setTokenRealm(tokenRealm);
+            response.setTokenRealm(null);
```

Comment:
This is actually for CXF realms, which have nothing to do with our DDF realms. We aren't currently using CXF realms at the moment, but removing all this code is going to make it so that we actually cannot use CXF realms. Are we sure that we want to remove all the code above this?